docs/source/examples/imperative-array.rst

```diff
@@ -0,0 +1,49 @@
+Simple parallel example
+=======================
```

Comment:
Perhaps a more informative title.  This ends up populating TOC and (maybe) sidebars